### PR TITLE
added event api tests

### PR DESCRIPTION
**Pull request by REKYAR**

### CODE REVIEW

**Review by Xevi8X — APPROVED** [review]

**Comment by Xevi8X** [comment]
LGTM

File: Tests/EventApiControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+
+            // Act
+            var result = await _controller.PatchEvent(sessionToken, "999", patchData);
+
+            // Assert
+            Assert.IsType<StatusCodeResult>(result);
+            Assert.Equal(404, ((StatusCodeResult)result).StatusCode);
+        }
+    }
+}
```
Comment (Xevi8X) [inline Tests/EventApiControllerTests.cs:443]: Missing new line at the end of file

File: Backend/Controllers/EventOrganizerApi.cs [hunk 2]
```diff
@@ -189,7 +189,8 @@ public virtual async Task<IActionResult> LoginOrganizer([FromHeader][Required()]
             await _context.SaveChangesAsync();
 
             SessionResponseDTO dto = session.AsDto();
-            return StatusCode(200, dto);
+            //return StatusCode(200, dto);
+            return new ObjectResult(dto) { StatusCode = 200 };
```
Comment (Xevi8X) [inline Backend/Controllers/EventOrganizerApi.cs]: Dlaczego?

Comment (REKYAR) [reply Backend/Controllers/EventOrganizerApi.cs]: fixed

**Pull request merged**